vms/platformvm/block/builder/builder.go

```diff
@@ -192,9 +192,7 @@ func (b *builder) ShutdownBlockTimer() {
 // This method removes the transactions from the returned
 // blocks from the mempool.
 func (b *builder) BuildBlock(context.Context) (snowman.Block, error) {
-	b.Mempool.DisableAdding()
```

Comment:
Do we need to add some locking to the mempool now? I think there are a number of internal fields modified on the call to `Add`, for example?